Extension/Configuration/Models/SorterConfigurationSave.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace YAPO.Configuration.Models
+{
+    [UsedImplicitly]
+    public class SorterConfigurationSave
+    {
+        public string Id;
+        public DateTime LastSaved;
+        public SorterConfiguration Party;
+        public SorterConfiguration Other;
```

Comment:
So the file itself does not store which save it is related to? May make sense to also store that

Reply:
Yeah, `Id` is the save name

Reply:
Then it may make sense to name it accordingly? This way someone needs to know that the ID is the save name for it to make sense.

Reply:
Yep good idea